tox.ini

```diff
@@ -0,0 +1,13 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+skipsdist = True
+envlist = py35
+
+[testenv]
+deps =
+    -rrequirements.txt
+commands = py.test
```

Comment:
Is that really the case? I've always used pip to override system-wide binaries with more recent ones in virtualenvs, so I'd be extremely surprised if pip failed to install the py.test executable like that. (I would file it as a bug with pip, if that happened?)